webapp/frontend/lib/settings/seed_screen.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:get_10101/settings/seed_words.dart';
+import 'package:get_10101/settings/settings_service.dart';
+import 'package:provider/provider.dart';
+
+class SeedScreen extends StatefulWidget {
+  const SeedScreen({super.key});
+
+  @override
+  State<SeedScreen> createState() => _SeedScreenState();
+}
+
+class _SeedScreenState extends State<SeedScreen> {
+  bool checked = false;
+  bool visibility = false;
+
+  List<String>? phrase;
+
+  @override
+  void initState() {
+    context.read<SettingsService>().getSeedPhrase().then((value) => setState(() => phrase = value));
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Padding(
+        padding: const EdgeInsets.only(top: 20, left: 10, right: 10),
+        child: Column(
+          children: [
+            Container(
+              margin: const EdgeInsets.all(10),
+              child: Center(
+                child: RichText(
+                    text: const TextSpan(
+                        style: TextStyle(color: Colors.black, fontSize: 18),
+                        children: [
+                      TextSpan(
+                          text:
+                              "The recovery phrase (sometimes called a seed), is a list of 12 English words. It allows you to recover full access to your funds if needed\n\n"),
+                      TextSpan(
+                          text: "Do not share this seed with anyone. ",
+                          style: TextStyle(fontWeight: FontWeight.bold)),
+                      TextSpan(
+                          text:
+                              "Beware of phising. The developers of 10101 will never ask for your seed.\n\n"),
+                      TextSpan(
+                          text: "Do not lose this seed. ",
+                          style: TextStyle(fontWeight: FontWeight.bold)),
+                      TextSpan(
+                          text:
+                              "Save it somewhere safe (not on this phone). If you lose your seed and your phone, you've lost your funds."),
```

Comment:
We don't run this on the phone. 

```suggestion
                              "Save it somewhere safe. If you lose your seed and your phone, you've lost your funds."),
```